examples/web-authn-oauth.py

```diff
@@ -42,25 +42,28 @@ class MyOAuthSecuredApplication(asab.Application):
 		# Create a dedicated web container
 		container = asab.web.WebContainer(websvc, 'example:oauth')
 
-		# Select OAuth providers (only GitHub in our case)
+		# Load the OAuth module
+		self.add_module(asab.web.authn.oauth.Module)
+		oauth_client_service = self.get_service("asab.OAuthClientService")
 
-		oauth_methods = [asab.web.authn.oauth.GitHubOAuthMethod()]
+		# Select OAuth providers (only GitHub in our case)
+		oauth_client_service.append_method(asab.web.authn.oauth.GitHubOAuthMethod())
 
 		# Add middleware for authentication via oauth2
 		container.WebApp.middlewares.append(
 			asab.web.authn.authn_middleware_factory(
 				self,
 				"oauth2client",
-				methods=oauth_methods,
+				oauth_client_service=oauth_client_service,
 			)
 		)
 
+		# Register useful OAuth endpoints
+		oauth_client_service.configure(container=container)
```

Comment:
`asab.web.authn.authn_middleware_factory` can be added to a container in the oauth_client_service.configure()` as well, correct?